storage/scylla_api.c

```diff
@@ -0,0 +1,616 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "scylla_api.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define SCYLLA_API_LOGGER "SCYLLA_API"
+
+static logger_id_t scylla_api_logger_id;
+
+#define SCYLLA_LOG_ERROR(fmt, ...) \
+  logger_helper_print(scylla_api_logger_id, LOGGER_ERR, "[%s:%d:]" fmt, __func__, __LINE__, __VA_ARGS__)
+#define SCYLLA_LOG_DEBUG(fmt, ...) \
+  logger_helper_print(scylla_api_logger_id, LOGGER_DEBUG, "[%s:%d:]" fmt, __func__, __LINE__, __VA_ARGS__)
+
+void scylla_api_logger_init() { scylla_api_logger_id = logger_helper_enable(SCYLLA_API_LOGGER, LOGGER_DEBUG, true); }
+
+int scylla_api_logger_release() {
+  logger_helper_release(scylla_api_logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    log_critical(scylla_api_logger_id, "[%s:%d] Destroying logger failed %s.\n", __func__, __LINE__, SCYLLA_API_LOGGER);
+    return EXIT_FAILURE;
+  }
+  return 0;
+}
+
+typedef struct Select_Where_ {
+  cass_byte_t* bundle;
+  cass_byte_t* address;
+} Select_Where;
+
+typedef enum { WITH_BUNDLE = 0, WITH_BUNDLE_AND_ADDRESS, NUM_OF_SELECT_METHOD } Select_Way;
```

Comment:
How about renaming to `SelectMethod`?

Reply:
Method sounds better than Way.
Btw, It is good to modify all the function name and struct name with mixed cases to follow the Google coding style?
For example,  `Select_Where` to `SelectWhere` or 
`insert_transaction_into_bundleTable` to `InsertTransactionIntoBundleTable`

Reply:
I defer to your perspective on consistent style.

Reply:
dicussed with Jserv, we prefer to use `select_method_t`